lib/parity/environment.rb

```diff
@@ -50,11 +50,23 @@ def restore
         Backup.new(
           from: arguments.first,
           to: environment,
-          additional_args: arguments.drop(1).join(" ")
+          additional_args: additional_restore_arguments,
         ).restore
       end
     end
 
+    def additional_restore_arguments
+      (arguments.drop(1) + [restore_confirmation_argument]).
+        compact.
+        join(" ")
+    end
+
+    def restore_confirmation_argument
+      if !["production", "development"].include?(environment)
```

Comment:
Development will never need that argument since it's not a Heroku environment, but in the current organization of this code, `#restore_confirmation_argument` would never be called anyway. I'll change that.

@croaky has expressed a strong concern about overwriting production data by accident (hence why you can't restore to production), which I think is a good idea. Over a lot of time using parity I've come to the conclusion that anything outside production should be considered volatile and fair-game. An environment variable would be a great compromise if there's concern about blowing away staging data.